pkg/provider/pulumi/gcp/gateway.go

```diff
@@ -228,6 +308,18 @@ func gcpOperation(op *openapi2.Operation, urls map[string]string) *openapi2.Oper
 		return nil
 	}
 
+	if s, ok := op.Extensions["x-nitric-security"]; ok {
+		secName, isString := s.(string)
+
+		if isString {
```

Comment:
minor: if it is not a string? I guess if it is nil, that is fine. but do you want to warn if it isn't?